core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/wikipedia/Redirect.scala

```diff
@@ -12,7 +12,7 @@ object Redirect
     private def mappings_redirects = Set("#REDIRECT")
     private def commons_redirects = Set("#REDIRECT")
-    private def wikidata_redirects = Set("#REDIRECT")
+    private def wikidata_redirects = Set("\"redirect\":")
```

Comment:
with this, I think we lose the redirects from pages that are not wikidata items but this is not important imo
(might worth making a comment on this though)

Reply:
Why do you think that we will lose redirects from pages that are not Wikidata items?